### PR TITLE
Add some content to README.md

### PR DESCRIPTION
**Pull request by marcua**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
 # Orchestra
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/unlimitedlabs/orchestra?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+Test status [![Circle CI](https://circleci.com/gh/unlimitedlabs/orchestra.svg?style=svg)](https://circleci.com/gh/unlimitedlabs/orchestra)
+
+Chat with the team [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/unlimitedlabs/orchestra)
+
+Orchestra is an Apache 2-licensed system for coordinating project teams of experts and machines.  Projects can include everything from design teams working on client delivery to newsrooms bringing together reporters, photographers, and editors for a story.  Orchestra supports expert review, so an expert working on any task in a project can get feedback and iteratively refine their work.  It also brings machines and automation onto projects: a crawler can collect content from the web before an expert combs through it, or a classifier can filter out bad data so an analyst spends less time in the noise.
```
Comment (thisisdhaas) [inline README.md]: Pick 80-char lines or 1-line-per-sentence and stick with it please!

AM: done

Comment (thisisdhaas) [inline README.md]: Just "Apache-licensed" is fine, the 2 is unnecessary. Actually do we even need to call out the Apache bit at all?

AM: done, created `License` section

Comment (thisisdhaas) [inline README.md]: "design teams working on client delivery" -- the 'client delivery' bit sounds a little stuffy.

Also maybe need some text on extensibility/how to build around Orchestra here: "New workflows can be added with some simple python glue and an html interface."

AM: done for both

Comment (thisisdhaas) [inline README.md]: 'Orchestra supports expert review' will be opaque to most. Something like "Orchestra workflows can assign senior experts to review other experts in order provide feedback and iteratively refine the work"?

AM: done

File: README.md [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+
+To learn more about Orchestra, including some examples, a getting started guide, and various ways to reach the Orchestra team, take a look at some of these resources:
```
Comment (thisisdhaas) [inline README.md]: "to reach out to the Orchestra team"

AM: done

File: README.md [hunk 3]
```diff
@@ -1,6 +1,22 @@
...
+
+* [A blog post introducing Orchestra v0.1.0 to the world]()
+* [Getting started with Orchestra in 5 minutes]()
+* [Orchestra website](http://orchestra.unlimitedlabs.com)
+* [Chat with the Orchestra team](https://gitter.im/unlimitedlabs/orchestra)
+* [Subscribe to the Orchestra mailing list]()
+
```
Comment (thisisdhaas) [inline README.md:28]: Also link to your strata slides once they're available.

AM: done

File: README.md [hunk 4]
```diff
@@ -1,6 +1,39 @@
 # Orchestra
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/unlimitedlabs/orchestra?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+Test status [![Circle CI](https://circleci.com/gh/unlimitedlabs/orchestra.svg?style=svg)](https://circleci.com/gh/unlimitedlabs/orchestra)
+
+Chat with the team [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/unlimitedlabs/orchestra)
+
+Orchestra is a system for coordinating project teams of experts and machines.
+Projects can include everything from design teams working on a client's
+project
```
Comment (thisisdhaas) [inline README.md:9]: weird space

File: README.md [hunk 5]
```diff
@@ -1,6 +1,39 @@
...
+to newsrooms bringing together reporters, photographers, and editors for a
+story.  In Orchestra workflows, you can assign senior experts to review other
+experts in order to provide feedback and iteratively refine the work.  It also
```
Comment (thisisdhaas) [inline README.md]: 'It' => Orchestra

AM: done

File: README.md (hunk 4 above)
Comment (marcua) [inline README.md:9]: i will not both do 80 char lines and rewrap on edit :)

**Comment by thisisdhaas** [comment]
We'll have to work on your 80-chars foo, but LGTM.

**Pull request merged**